roles/openshift_cluster_monitoring_operator/templates/cluster-monitoring-operator-config.j2

```diff
@@ -5,6 +5,9 @@ metadata:
   namespace: {{ openshift_cluster_monitoring_operator_namespace }}
 data:
   config.yaml: |+
+{% if openshift_cluster_monitoring_extra_targets -%}
+{{ openshift_cluster_monitoring_extra_targets | to_nice_yaml(indent=2) | indent(4, True) }}
```

Comment:
This config does not allow doing this, and beyond that the cluster-monitoring stack fundamentally does not support adding any additional targets. Modifying it voids any type of support for it.